packages/cli/commands/project/migrateApp.js

```diff
@@ -185,6 +191,12 @@ exports.handler = async options => {
         { includesRootDir: true, hideLogs: true }
       );
 
+      trackCommandMetadataUsage(
+        'migrate-app',
+        { projectName, appId, status, preventProjectMigrations, listingInfo },
```

Comment:
iirc you're not able to include objects in tracking events. For listing info you might be better off just using a boolean like `isListed`

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1127/commits/b9f3bd85d7238c6c72af57fdb90fbdd283208361